src/Chapter07/Listing07.29.PropertyPatternMatching.cs

```diff
@@ -0,0 +1,50 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter07.Listing07_29;
+
+
+#region INCLUDE
+public record class Employee
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public string Role { get; set; }
+
+    public Employee(int id, string name, string role) =>
+        (Id, Name, Role) = (id, name, role);
+}
+
+
+public class ExpenseItem
+{
+    public int Id { get; set; }
+    public string ItemName { get; set; }
+    public decimal CostAmount { get; set; }
+    public DateTime ExpenseDate { get; set; } = DateTime.Now;
```

Comment:
Default value appears to always get overridden in the constructor.
```suggestion
    public DateTime ExpenseDate { get; set; }
```